src/main/java/com/uber/jenkins/phabricator/PhabricatorNotifier.java

```diff
@@ -232,6 +248,31 @@ private String getCoverageComment(Ratio lineCoverage, UberallsClient uberalls, D
         }
     }
 
+    /**
+     * Attempt to read a remote comment file
+     * @param build the build
+     * @param logger the logger
+     * @return the contents of the string
+     * @throws InterruptedException
+     */
+    private String getRemoteComment(AbstractBuild<?, ?> build, PrintStream logger, String commentFile) throws InterruptedException, IOException {
+        if (CommonUtils.isBlank(commentFile)) {
+            logger.println("[comment-file] no comment file configured");
+            return null;
+        }
+
+        FilePath workspace = build.getWorkspace();
+        FilePath[] src = workspace.list(commentFile);
+        if (src.length == 0) {
+            logger.println("[comment-file] no files found by path: '" + commentFile + "'");
+            return null;
+        }
+        if (src.length > 1) {
+            logger.println("[comment-file] Found multiple matches. Reading first only.");
```

Comment:
Is this code reachable? Does the plugin need to take in some kind of wildcard in order for us to get here?